src/tests/api/cat.test.ts

```diff
@@ -6,9 +6,20 @@
 import path from 'path';
 import { ApiProviderFactory } from '../../providers/factory';
 import { QARunner } from '../../runners/qa/qa_runner';
+import { OpenSearchTestIndices } from '../../utils/indices';
 
 const provider = ApiProviderFactory.create();
-const runner = new (class CatIndicesRunner extends QARunner {})(provider);
+
+const runner = new (class CatIndexRunner extends QARunner {
+    protected async beforeAll(clusterStateId: string): Promise<void> {
+      if (clusterStateId !== 'spider') {
+        throw new Error('unexpected cluster state id');
+      }
+      await OpenSearchTestIndices.delete('spider');
```

Comment:
no need to delete? since it won't change and it takes a while to create

Reply:
Sure, was mostly copying from what you had earlier :)